specification/maps/resource-manager/Microsoft.Maps/preview/2021-07-01-preview/maps-management.json

```diff
@@ -0,0 +1,1197 @@
+            "$ref": "#/definitions/MetricSpecification"
+          }
+        }
+      }
+    },
+    "MetricSpecification": {
+      "description": "Metric specification of operation.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of metric specification."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of metric specification."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Display description of metric specification."
+        },
+        "unit": {
+          "type": "string",
+          "description": "Unit could be Count."
+        },
+        "dimensions": {
+          "description": "Dimensions of map account.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Dimension"
+          }
+        },
+        "aggregationType": {
+          "type": "string",
+          "description": "Aggregation type could be Average."
+        },
+        "fillGapWithZero": {
+          "type": "boolean",
+          "description": "The property to decide fill gap with zero or not."
+        },
+        "category": {
+          "type": "string",
+          "description": "The category this metric specification belong to, could be Capacity."
+        },
+        "resourceIdDimensionNameOverride": {
+          "type": "string",
+          "description": "Account Resource Id."
+        }
+      }
+    },
+    "Dimension": {
+      "description": "Dimension of map account, for example API Category, Api Name, Result Type, and Response Code.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Display name of dimension."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of dimension."
+        }
+      }
+    },
+    "MapsAccountProperties": {
+      "description": "Additional Map account properties",
+      "type": "object",
+      "properties": {
+        "uniqueId": {
+          "description": "A unique identifier for the maps account",
+          "type": "string",
+          "readOnly": true
+        },
+        "disableLocalAuth": {
+          "description": "Allows toggle functionality on Azure Policy to disable Azure Maps local authentication support. This will disable Shared Keys authentication from any usage.",
+          "type": "boolean",
+          "default": false
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the Map account resource.",
+          "readOnly": true
+        },
+        "linkedResources": {
+          "type": "object",
+          "$ref": "#/definitions/LinkedResources",
+          "description": "Sets the resources to be used for Managed Identities based operations for the Map account resource."
+        }
+      }
+    },
+    "LinkedResources": {
```

Comment:
as we reviewed today @jorgecotillo, I've updated LinkedResources to be an array of an object.  We lose the ability to patch an individual resource. However we gain type safety with a path forward to add additional properties in the future. We will leave out `type` as this can be resolved by the `id` in the RP.

Take a look if you have a chance, thanks again!

Reply:
Looks good. I think the example files need to be updated to reflect this change too.

Reply:
Thanks @filizt, example files are updated. If it looks good, feel free to update the PR tags.